worker-store-fs/src/main/java/com/hpe/caf/worker/datastore/fs/FileSystemDataStore.java

```diff
@@ -79,7 +79,27 @@ public FileSystemDataStore(final FileSystemDataStoreConfiguration config)
 
         LOG.debug("Initialised");
     }
-
+    
+    private static boolean pathDoesNotExist(final Path path)
```

Comment:
I must not be explaining this Jira correctly.  You've just replaced the `Files.exists` call with a method which has exactly the same problem.

Reply:
Resolved with [this](https://github.com/WorkerFramework/worker-framework/pull/128#issuecomment-840929683) implementation.